.github/workflows/test_and_deploy.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           python --version
           conda install -y pyopencl
           python -m pip install --upgrade pip
-          pip install setuptools wheel pytest pytest-cov pytest-benchmark dask
+          pip install setuptools wheel pytest pytest-cov pytest-benchmark dask networkx python-igraph
```

Comment:
`python-igraph` is now simply `igraph` on PyPI; `python-igraph` will keep on working for a while but it's only a dummy package that depends on `igraph` now. You should install `igraph` directly from PyPI